### PR TITLE
Withdraw MGN only when the lock is expired

### PR DESCRIPTION
**Pull request by anxolin**
The previous check for detecting if the MGN is unlocked was not working, the call was not failing and the transaction was

Instead, this PR includes a different check: It verifies the withdraw time of the locked MGN

### CODE REVIEW

File: scripts/participateInAuction.js [hunk 1]
```diff
@@ -96,20 +99,24 @@ async function unlockMgnAndTokens({
 }
 
 
-async function claimMgn({ poolNumber, pool, state, infoAuction }) {
+async function claimMgn({ poolNumber, pool, mgn, state, infoAuction, debugAuction }) {
   // We could check the expire time easily of the MGN, but we will take the simple approach of
```
Comment (bh2smith) [inline scripts/participateInAuction.js]: can now be removed.

Comment (bh2smith) [reply scripts/participateInAuction.js]:
```suggestion
```

File: scripts/participateInAuction.js [hunk 2]
```diff
@@ -96,20 +99,24 @@ async function unlockMgnAndTokens({
...
   // executing "withdrawUnlockedMagnoliaFromDx" if the call succeeds
```
Comment (bh2smith) [inline scripts/participateInAuction.js]:
```suggestion
```

Comment (anxolin) [reply scripts/participateInAuction.js]: Thanks for the review. I removed also the other comment. I think it was not necessary either

**Pull request merged**